main.go

```diff
@@ -110,6 +112,12 @@ func startServer(ctx context.Context, d *daemon, tcpListener, metricsUsername, m
 			}
 		}
 
+		if ipniIndexer == "" {
+			ipniIndexer = DEFAULT_IPNI_INDEXER
+		}
+
+		FindIPNIProviders(ctx, cidStr, ipniIndexer)
```

Comment:
This can probably move to `runCidCheck` which is the "CID without provider/multiaddr check", since in the other case you are only testing with the user provided multiaddr.

Reply:
Done